### ISSUE TITLE
Data-scroll-call and resizing the window

### ISSUE THREAD

**Issue by LaurieVince** [locomotivemtl/locomotive-scroll#100]
Hey everybody!

I found a small bug about the **data-scroll-call** method.
If my section that has this method is in-view when I'm resizing the window my function is called so many times ! 

[Here's a tiny example of my problem](https://codesandbox.io/s/nervous-benz-3xv3b?fontsize=14&hidenavigation=1&theme=dark)

I want to add a gsap animation on every scroll-call, and if i resize the window, the animation is looping...so this issue is important to me. 
**So, is this possible to remove the call in resizing window ?**

Thanks guys! 

Laurie

**Comment by imcorentin** [comment]
Hi Laurie,

If you want to animate elements with GSAP and LS, I recommend you to use a timeline (and set it paused), like this :
`const tl = new gsap.timeline({ paused: true })`

Then, you can simply play your timeline directly inside your function :  
```
scroll.on("call", value => {
  if (value === "myScrollDataValue") {
    tl.play();
   }
 });
```

Have fun !

**Comment by quentinhocde** [comment]
Hello,

It's fixed in the 3.5.1 release. On update, we check if the element was in viewport, and we set the right value to the `inView`.

Thanks!
